Sources/Filters.swift

```diff
@@ -89,6 +89,11 @@ struct StringFilters {
     }
   }
 
+  static func camelToSnakeCase(_ value: Any?) throws -> Any? {
+    guard let string = value as? String else { throw FilterError.invalidInputType }
+    return try snakecase(string).lowercased()
```

Comment:
Unfortunately the Wikipedia page doesn't give any definitive rules for upper snake case. For instance, as the StringFilter.snakecase() function is currently written, URLChooser becomes URL_Chooser, not Url_Chooser. I couldn't really find anything that said that'd be right or wrong, so I think for now I'll just stick with that implementation. The filter itself will take one optional boolean parameter for lowercasing, defaults to true.